packages/utilities/fast-web-utilities/src/key-codes.ts

```diff
@@ -1,6 +1,6 @@
 /*
  * Key Code values
- * @deprecated - use individual keycode exports
+ * @deprecated - KeyCodes are deprecated, use individual string key exports
```

Comment:
Two things here:

1. It doesn't look like this is actually jsdoc / tsdoc syntax, so the deprecation comments are getting lost to the d.ts files when this is built. Can you change:

```ts
/*
 * ...
 */
```

to this:
```ts
/**
 * ....
 */
```
You should see the `@deprecated` flag light up with syntax highlighting. Once you do, you should see the `@deprecated` in the emitted d.ts files.

2. For the const exports, you're actually only deprecating `keyCodeAlt`, not the entire block. I know it's a pain, but would you add the `@deprecated` to *every* export so that authors will see the warning in their editors?

Reply:
Thanks!